src/main.ts

```diff
@@ -55,7 +50,7 @@ class KnxFrontend extends knxElement {
 
     return html`
       <ha-app-layout>
-        <app-header fixed slot="header">
+        <app-header fixed condenses slot="header">
           <app-toolbar>
             <ha-menu-button .hass=${this.hass} .narrow=${this.narrow}></ha-menu-button>
             <div main-title>KNX UI</div>
```

Comment:
shouldn't this use the translation value?
```suggestion
            <div main-title>${localize(this.hass.language, "title")}</div>
```